x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/pages/empty.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { EuiButton, EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
+import { SecurityPageName } from '@kbn/deeplinks-security';
+import { css } from '@emotion/react';
+import { useNavigation } from '../../../common/lib/kibana';
+import * as i18n from './translations';
+
+export const EmptyMigrationRulesPage = () => {
+  const { navigateTo } = useNavigation();
+
+  const navigateToStartMigrationCallback = useCallback(() => {
+    navigateTo({ deepLinkId: SecurityPageName.landing, path: 'siem_migrations#start' });
```

Comment:
Can we use the onboarding topics and cards enums to build the `path`, instead of hardcoding the string, they are [here](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/onboarding/constants.ts).

We'll probably need to update the `start` to `rules_start` or something before introducing dashboard migrations.

Reply:
thanks.. will add that.